frontend/src/pages/transactions/main/Main.tsx

```diff
@@ -71,6 +71,7 @@ const columns = [
     header: () => <Label variant="medium">STATUS</Label>,
     cell: (info) => (
       <div>
+        {/* TODO(milestone-5): Display transaction status icon */}
```

Comment:
@MrakJakob You can go ahead and add logic for rendering status icons when you find the time. I added a task for this here: https://www.notion.so/flowser/Update-transaction-status-icons-152f6301c1b84a36a4160aca6a5338f9